x-pack/plugins/alerting/server/application/rule/methods/create/create_rule.ts

```diff
@@ -60,6 +61,12 @@ export async function createRule<Params extends RuleParams = never>(
 
   try {
     createRuleDataSchema.validate(data);
+    if (data.enabled) {
```

Comment:
we might want to let the user create the rule but disable the rule when they pass the schedule limit.

Reply:
https://github.com/elastic/kibana/pull/164791/commits/74a53604d4cad622dc43ad41d53d08f8cddcb859